docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -117,6 +121,15 @@ CREATE TABLE student_copy USING CSV
 --Omit the USING clause, which uses the default data source (parquet by default)
 CREATE TABLE student (id INT, name STRING, age INT);
 
+--Use parquet data source with parquet storage options
+--The columns 'id' and 'name' enable the bloom filter during writing parquet file,
+--column 'age' does not enable
+CREATE TABLE student_parquet(id INT, name STRING, age INT) USING PARQUET
+    OPTIONS (
+      'parquet.bloom.filter.enabled'='true',
```

Comment:
Other examples of OPTIONS seem different, and would look more like:
`parquet.bloom.filter.enabled "true",`
It seems. Is this right and/or should we standardize?

Reply:
yes, both of them are fine, see:
https://github.com/apache/spark/blob/59365703c51a36efca5390b7535489a839f51659/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4#L365-L395

I just follow the `TBLPROPERTIES` format in this docs.